app/models/cherrypick_task.rb

```diff
@@ -217,11 +221,11 @@ def control_positions(batch_id, num_plate, num_free_wells, num_control_wells)
 
```

Comment:
Yes, sorry, my mistake. I think the name `num_free_wells` is not the right name.
The value is coming from PlateTemplate.size, which it is giving the size of the labware (96,384, etc...) not the number of free wells in the labware. 
I think we could rename it to something more correct (total_wells, num_wells, ...)